src/datatypes/datetime.coffee

```diff
@@ -384,3 +395,77 @@ module.exports.DateTime = class DateTime
       fieldsToRemove = DateTime.FIELDS.slice(fieldIndex + 1)
       reduced[field] = null for field in fieldsToRemove
     reduced
+
+normalizeMillisecondsFieldInString = (string, matches) ->
+  msString = matches[14]
+  # TODO: verify we are only removing numeral digits
```

Comment:
Can you add a skipped test for this situation? We are more likely to implement this check down the line if we have a test for it.